### PR TITLE
vsh.self: Fix execution on default settings

### PR DESCRIPTION
**Pull request by elad335**
Due to a bug, a special debug setting was required to be toggled which has nothing to do with VSH, to run VSH. This is fixed with this PR.

### CODE REVIEW

**Comment by RainbowCookie32** [comment]
VSH runs without Debug Console mode now 👍🏼 

image

File: rpcs3/Emu/Cell/PPUModule.cpp [hunk 1]
```diff
@@ -1788,17 +1788,16 @@ bool ppu_load_exec(const ppu_exec_object& elf)
 		mem_size = 0xC800000;
 	}
 
-	if (g_cfg.core.debug_console_mode)
+	if (g_ps3_process_info.get_cellos_appname() == "vsh.self"sv)
```
Comment (Megamouse) [inline rpcs3/Emu/Cell/PPUModule.cpp:1791]: This can be put as first check of the upper if else section then

**Comment by Megamouse** [comment]
What if i boot vsh with debug mode now? Will it still work?

File: rpcs3/Emu/Cell/PPUModule.cpp (hunk 1 above)
Comment (elad335) [reply rpcs3/Emu/Cell/PPUModule.cpp]: Done.

**Comment by elad335** [comment]
Yes.

**Pull request merged**